qemu/tests/cfg/fio_perf.cfg

```diff
@@ -0,0 +1,57 @@
+- fio:
+    only x86_64
+    type = fio_perf
+    numa_node = -1
+    block_size = "4k 16k 64k 256k"
+    format = False
+    iodepth = "1 8 64"
+    threads = "16"
+    rw = "read write randread randwrite randrw"
+	Linux:
+        no ide
+        pattern = ".*[read|write].*bw=(\d+(?:\.\d+)?[\w|\s]B/s),\siops=(\d+).*"
+        order_list = "Block_size Iodepth Threads BW(MB/S) IOPS Latency(ms) Host_CPU BW/CPU KVM_Exits Util%"
+        guest_ver_cmd = "uname -r"
+        compile_cmd = "make && make install"
+        pre_cmd = "i=`/bin/ls /dev/[vs]db` && mkfs.xfs $i > /dev/null; partprobe; umount /mnt; mount $i /mnt"
+        drop_cache = "sync && echo 3 > /proc/sys/vm/drop_caches"
+        guest_result_file = /tmp/fio_result
+        fio_cmd = "fio --rw=%s --bs=%s --iodepth=%s --runtime=1m --direct=1 --filename=/mnt/%s --name=job1 --ioengine=libaio --thread --group_reporting --numjobs=%s --size=512MB --time_based --output=/tmp/fio_result &> /dev/null"
+	Windows:
```

Comment:
All indent is correct on my side before push, I don't know yet why it is changed after push. Checking....